tests/test_international_days.py

```diff
@@ -0,0 +1,455 @@
+from datetime import date
```

Comment:
That can be a bit of a problem - if we change the text we should also change the tests?
I would prefer something more agile

Reply:
I used it exactly as zohar used in her quotes ticket

Reply:
The comment is on the whole file, not specifically on this line.

Reply:
I don't understand, where do I change the text?

Reply:
The problem is with the way the tests are designed.

We need to design them in such a way that if, in the future, we will add new international days, or remove old days - the tests will still work.

Reply:
Please fix this in order to merge this ticket.

Reply:
fixed